fuzzers/frida_libpng/build.rs

```diff
@@ -26,7 +26,7 @@ fn build_dep_check(tools: &[&str]) {
 
 fn main() {
     if cfg!(windows) {
-        println!("cargo:warning=Skipping libpng frida example on Windows");
+        println!("cargo:warning=You have to compile the hareness manually on Windows.");
```

Comment:
Shouldn't compiling be possible in build.rs? How to do it otherwise?

Reply:
Yes the commands for building is pretty much the same on windows if you use msys2.
I'll try to adjust build.rs for msys2+mingw